clang/include/clang/Basic/Builtins.h

```diff
@@ -309,7 +309,10 @@ enum BuiltinTemplateKind : int {
   BTK__make_integer_seq,
 
   /// This names the __type_pack_element BuiltinTemplateDecl.
-  BTK__type_pack_element
+  BTK__type_pack_element,
+
+  /// This names the __type_list_dedup BuiltinTemplateDecl.
```

Comment:
This name seems inconsistent with the previous builtin: using the `__type_pack` prefix for both would be more uniform.

Reply:
I think we've decided somewhere that all new builtin functionality should have a `__builtin_` prefix. The main reason was to avoid naming conflicts. While I don't think that's a huge issue here, we should probably be consistent in that regard. So this should probably be `__builtin_type_pack_dedup`.

Reply:
Thanks, I don't actually know why I didn't pick `type_pack` prefix in the first place. I think I was confused when I started and thought that `type_pack_element` was more different than it ended up being.

It's now `__builtin_type_pack_dedup`, as suggested.